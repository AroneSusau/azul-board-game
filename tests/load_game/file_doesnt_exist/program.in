2
i_dont_exist
