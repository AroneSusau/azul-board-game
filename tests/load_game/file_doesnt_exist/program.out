
      db    8888P 8    8 8    
    dPYb     dP  8    8 8    
  dPwwYb   dP   8b..d8 8    
dP    Yb d8888 `Y88P' 8888

Info: No seed argument detect, using seed 0 instead.
Enter a number
1. Start Game
2. Load Game
3. Credits
4. Exit
> Enter the filename from which load a game
> Error: Save file i_dont_exist does not exist.
> See ya :D
